src/components/employeeCard/EmployeeCard.tsx

```diff
@@ -30,38 +30,58 @@ export default function EmployeeCard({
           <div className={styles.employeeImage}>
             <Image
               src={employee.imageUrl}
-              alt=""
+              alt={employee.name}
               style={{ objectFit: "cover" }}
               fill={true}
             />
           </div>
         </Link>
```

Comment:
Maybe we should remove the link to the mini-cv for now?

Reply:
I think it adds value to be able to link to contact information for a particular person 🤔

Reply:
Agreed that we should have links to contact info, but do we need link to the mini-cv page?

Reply:
I mean contact page. As in "Contact this person http://localhost:3000/no/folk/mn"